Dockerfile

```diff
@@ -5,5 +5,8 @@ COPY . /app
 WORKDIR /app
 RUN sed -i'' 's/archive\.ubuntu\.com/us\.archive\.ubuntu\.com/' /etc/apt/sources.list
 RUN apt-get clean -y && rm -r /var/lib/apt/lists/* -vf && apt-get clean -y && apt-get update -y && apt-get upgrade -y && apt-get install sudo -y
-RUN ./install.sh --no-flash
+
+# For some reasons, the build-extension.sh script does not work as root. We create another user and allow him to run sudo without a password. Then we run the install.sh script as openwpm, which calls build-extension.sh.
```

Comment:
Please use gender-neutral pronouns. s/him/it/

Reply:
Fixed